### PR TITLE
fix(fmgc/mcdu): various small issues

### LINKED ISSUE #7015: Go around/missed approach - FMC flight plan bug
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-04-07T06:10:36+00:00",
  "ref": "master",
  "sha": "a2906c70d9f97f6effc8276357a025fc3b700fc8",
  "actor": "tracernz",
  "event_name": "manual",
  "pretty_release_name": "master:a2906c70"
}
```

### Describe the bug

On final approach, being told to go around by ATC, once immediate steps taken (thrust, lift nose, gear, flaps etc) the flight plan page of the FMC shows four square boxes.  

To rectify need to go back to init page, reinsert start/finish airport code which re-initialises the flight plan page so that the destination can be selected and the new approach, via etc can be inserted.

### Expected behavior

Flight plan page on FMC should continue to show destination airport and/or alternate flight plan / missed approach rather than showing four white square boxes.

### Steps to reproduce

1. ATC calls out go around
2. TOGA thrust, raise nose, gear and flaps to 1
3. Fly runway heading to 5,000 feet (or safe alt)
4. Declare missed approach on ATC window
5. Switch ATC frequency as direct
6. Await new approach instructions
7. Go to FMC flight plan page to input new approach but cannot as FMC looks like it has crashed (four white square boxes shown).

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by tracernz**
Fixes #7015

## Summary of Changes

- Fix crash of f-pln page after sequencing last leg (e.g. go around).
- Fix to waypoint bearing sent to the ND (make it magnetic).
- Fix new cruise alt message (not changed during previous refactor).

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- Check that to waypoint bearing at the top right of the ND lines up with the ND bearing ring.
- Have F-PLN page open during go-around and make sure it doesn't crash on crossing the runway threshold.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js [hunk 1]
```diff
@@ -675,7 +675,7 @@ class FMCMainDisplay extends BaseAirliners {
                 // This checks against the pilot defined cruise altitude and the automatically populated cruise altitude
                 if (this.cruiseFlightLevel !== this._cruiseFlightLevel) {
                     this._cruiseFlightLevel = this.cruiseFlightLevel;
-                    this.addNewMessage(NXSystemMessages.newCrzAlt.modifyMessage(this._cruiseFlightLevel * 100));
+                    this.addNewMessage(NXSystemMessages.newCrzAlt.getSetMessage(this._cruiseFlightLevel * 100));
```
Comment (derl30n) [inline flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:678]: WOW interesting that this was wrong the entire time lol, ` getSetMessage` is indeed the correct function

**Review by derl30n — APPROVED** [review]
LGTM

File: flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js (hunk 1 above)
Comment (tracernz) [reply flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js:678]: Even funnier, I fixed another one a while back but mustn’t have looked further. 😆

**Comment by mico975** [comment]
Quality Assurance Tester Report

Discord          : mico#3145
Object of testing: #7033
Tier of Testing  : 1
Date             : 11/04/2022

Testing Process: 
Check that to waypoint bearing at the top right of the ND lines up with the ND bearing ring. - **OK**
Have F-PLN page open during go-around and make sure it doesn't crash on crossing the runway threshold. - **OK**

Negatives:       
N/A

Testing Results: 
Passed

Conclusions: 
LGTM

Media:       
image

**Pull request merged**